src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/special/preferences/EditPreferencesPage.java

```diff
@@ -17,11 +16,7 @@
   @FindBy(css = "input#mw-input-wpemailaddress")
   private WebElement emailAddressInput;
 
-  public EditPreferencesPage(WebDriver driver) {
-    super(driver);
-  }
-
-  public EditPreferencesPage openEditingSection(String wikiURL) {
+  private EditPreferencesPage openEditingSection(String wikiURL) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused private "openEditingSection" method. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AUnusedPrivateMethod)

Reply:
it is used